src/DurableSDK/PowerShellServices.cs

```diff
@@ -107,13 +99,15 @@ public void tryEnablingExternalDurableSDK()
             else
             {
                 // Log that the module was not found in worker path
+                var workerPathContents = PowerShellWorkerConfiguration.GetString("PSModulePath");
```

Comment:
The PSModulePath is a string array, e.g., 
```powershell
PS C:\> $env:PSModulePath -split ";"
C:\Users\<userName>\OneDrive - Microsoft\Documents\PowerShell\Modules
C:\Program Files\PowerShell\Modules
c:\program files\powershell\7\Modules
C:\Program Files\WindowsPowerShell\Modules
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell
```

Reply:
@Francisco-Gamino: I'm not sure I understand the feedback here. Are you saying you'd like to format the log so that each directory in PSModulePath is in its own line instead of using a single one?

Reply:
This `PowerShellWorkerConfiguration.GetString("PSModulePath");` will return a string like this: `C:\Users\<user>\OneDrive - Microsoft\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules;C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell`, so you will have to format it to make it more readable.